samples/central-login/README.md

```diff
@@ -0,0 +1,39 @@
+# Central login
+
+In cases in which a centralized login application is desired, in contrast to an embedded login form, the SDK provides a method for leveraging an OAuth flow known as Authorization Code Flow (with PKCE). By using an option (login of "redirect") provided by `TokenManager` to request OAuth/OIDC tokens, when the user is not authenticated, the SDK will redirect the user to the configured login application with the ForgeRock platform. Once authenticated, the user will be redirected back to the original application to attain OAuth/OIDC tokens to complete the user flow.
+
+You can run this sample app with the `npm run start:central-login` command. Please [see the Getting Started instructions](#getting-started) for more details.
```

Comment:
If we are going to change the command, I'd rather align it with the other sample apps. For the sample React app, it seems we recommend this: `npx nx run reactjs-todo:serve`. I'd prefer consistency between all the same apps.

Also, and more importantly, if this command changes, we HAVE to update the official SDK docs.

Reply:
Cool, I think we can have a discussion then as a team for the best way to write the commands. For posterity I'll write them here

1. Alias as an npm command in root package.json (uses nx command under the hood)
2. Leverage npm workspaces and add a command in each app (would use nx under the hood)
3. (npx) nx serve <app-name>
4. (npx) nx run <app-name>:serve

We can then use this to align *all* apps.

Reply:
going with: Alias as an npm command in root package.json (uses nx command under the hood)

Reply:
I'm very happy with this solution - hiding the Nx stuff under the hood / throughout the repo, is probably best where the samples are concerned. I mentioned it in our discussion; it's one less thing to make a developer think "can I run that" or "what does that tool do?"